src/renderer/components/ft-list-video/ft-list-video.js

```diff
@@ -729,30 +736,6 @@ export default defineComponent({
       this.watchProgress = 0
     },
 
-    togglePlaylistPrompt: function () {
-      const videoData = {
-        videoId: this.id,
-        title: this.title,
-        author: this.channelName,
-        authorId: this.channelId,
-        description: this.description,
-        viewCount: this.viewCount,
-        lengthSeconds: this.data.lengthSeconds,
-      }
-
-      this.showAddToPlaylistPromptForManyVideos({ videos: [videoData] })
-
-      // Focus when prompt closed
-      this.addToPlaylistPromptCloseCallback = () => {
```

Comment:
Looks like there are still some watchers and an entry in `data` for this property, can those be removed too?